### PR TITLE
more gaming

### PR DESCRIPTION
**Pull request by Gailbear**

### CODE REVIEW

**Review by emtersoff — APPROVED** [review]

File: frontend/src/app/gaming/signup/signup.component.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <div class="signup-genius">
-Signup Genius things will go here.
+    <p>Check back in this space for more info about signups!</p>
+
+    <p>For now, you can also go to the #main-gaming-chat on Discord.</p>
```
Comment (clahey) [inline frontend/src/app/gaming/signup/signup.component.html:4]: Make this a link to discord?

File: frontend/src/app/gaming/tabletop/tabletop.component.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.link {
```
Comment (clahey) [inline frontend/src/app/gaming/tabletop/tabletop.component.scss:1]: + cursor:pointer;

File: frontend/src/app/map/hall-costumes/hall-costumes.component.html [hunk 3]
```diff
@@ -1 +1,8 @@
-<app-carousel [images]="images"></app-carousel>
```
Comment (clahey) [inline frontend/src/app/map/hall-costumes/hall-costumes.component.html:1]: This is already fixed? Merge main?

File: frontend/src/app/gaming/signup/signup.component.html (hunk 1 above)
Comment (Gailbear) [reply frontend/src/app/gaming/signup/signup.component.html:4]: good idea

File: frontend/src/app/map/hall-costumes/hall-costumes.component.html (hunk 3 above)
Comment (Gailbear) [reply frontend/src/app/map/hall-costumes/hall-costumes.component.html:1]: youch that was a mistake fixing

**Pull request merged**